docs/themes/thxvscode/layouts/apis/aggPackage.html

```diff
@@ -28,13 +36,19 @@ <h3>List of bundled packages</h3>
                     </thead>
                     <tbody>
                         {{ $children := $.Params.children }}
-                        {{ range $children }}
-                        {{ range where $packagePages "Params.package" . }}
-                        <tr>
-                            <td><a href="{{ .RelPermalink | safeURL }}" title="{{ .Title }}">{{ .Title }}</a></td>
-                            <td>{{ truncate 170 (.Params.Summary | markdownify) }}</td>
-                        </tr>
-                        {{ end }}
+                        <!-- Iterate through specified package name strings -->
+                            {{ range $children }}
+                            <!-- Find and iterate over the actual corresponding package documents of matching version
+                                (we should only find 1 match)-->
+                            {{ range where $packagesWithMatchingVersion "Params.package" . }} 
+                                <tr>
+                                    <td><a href="{{ .RelPermalink | safeURL }}" title="{{ .Title }}">{{ .Title }}</a></td>
+                                    <td>{{ truncate 170 (.Params.Summary | markdownify) }}</td>
+                                </tr>
+                            {{ else }}
```

Comment:
Previously, this logic failed silently if we didn't find a match. Now we will log an error if we don't find one.